### PR TITLE
Burn Card Parameter

### PR DESCRIPTION
**Pull request by aaswenson**
Add thermal power as a modifiable mcnp input parameter.

### CODE REVIEW

File: neutronics/mcnp_inputs.py [hunk 1]
```diff
@@ -31,20 +31,22 @@ class HomogeneousInput:
     """Write Homogeneous Input File.
     Class to write homogeneous MCNP burnup input files.
     """
-    
-    def __init__(self, radius, length, pnnl_mats, thick_refl=15):
+    kW_to_MW = 0.001
```
Comment (bam241) [inline neutronics/mcnp_inputs.py]: don't we have a file for physical constant ?

File: neutronics/mcnp_inputs.py [hunk 2]
```diff
@@ -31,20 +31,22 @@ class HomogeneousInput:
...
+    kW_to_MW = 0.001    
+    def __init__(self, radius, length, power, pnnl_mats, thick_refl=15):
         """Initialize geometric reactor parameters.
 
         Initialized Attributes:
         -----------------------
             z (float): reactor core height
             r (float): reactor core radius
             frac_fuel (float): fuel to coolant channel fraction
+            Q_therm (float): reactor thermal power
```
Comment (bam241) [inline neutronics/mcnp_inputs.py]: maybe it is just me, but it feels strange to call power `Q`... I am used to `P`

File: neutronics/mcnp_inputs.py [hunk 3]
```diff
@@ -31,20 +31,22 @@ class HomogeneousInput:
...
             matlib (MaterialLibrary): PyNE material library
         """
         self.z = length
         self.r = radius
         self.refl_t = thick_refl
+        self.Q_therm = power * self.kW_to_MW
```
Comment (bam241) [inline neutronics/mcnp_inputs.py:50]: Why user have to provide power in MW? why not the same as the stored one ?
Why storing the value in kW ? Why not W ? or MW ?

File: neutronics/mcnp_inputs.py (hunk 2 above)
Comment (bam241) [reply neutronics/mcnp_inputs.py]: maybe as you have some expectation on the unit, it should be mentioned in the docs!

Comment (gonuke) [reply neutronics/mcnp_inputs.py]: P is often for pressure...

Comment (gonuke) [reply neutronics/mcnp_inputs.py]: Should definitely list units in the docstring if they are required.

**Review by gonuke — COMMENTED** [review]
Other than @bam241's question about why not just enter the power in MW - this looks good to me...

**Comment by gonuke** [comment]
Thanks @aaswenson

**Pull request merged**